static/templates/filings/candidate.hbs

```diff
@@ -56,14 +56,28 @@
       <td class="panel__term">Ending cash on hand</td>
       <td class="panel__data">{{{currency cash_on_hand_end_period}}}</td>
     </tr>
+    {{#if net_contributions_period}}
     <tr>
       <td class="panel__term">Net contributions</td>
       <td class="panel__data">{{{currency net_contributions_period}}}</td>
     </tr>
+    {{else}}
+    <tr>
+      <td class="panel__term">Net contributions</td>
+      <td class="panel__data">{{{currency net_contributions_cycle_to_date}}}</td>
+    </tr>
+    {{/if}}
+    {{#if net_operation_expenditures_period}}
```

Comment:
So we used to show all financial summary information for reports on the website, when we were showing most recent report. I remember doing a through check to make sure we had everything. It looks like we don't show that information anymore on the website. 

I guess we can close this. I do wish all the data was accessible somewhere on the website, perhaps we could do a reports view in the future?